test/functional/mempool_package_limits.py

```
@@ -174,6 +176,73 @@ def test_desc_count_limits(self):
+
+        parent_signed = node.signrawtransactionwithkey(hexstring=rawtx, privkeys=self.privkeys)
+        assert parent_signed["complete"]
+        parent_tx = tx_from_hex(parent_signed["hex"])
+        parent_txid = parent_tx.rehash()
+        node.sendrawtransaction(parent_signed["hex"])
+
+        # Chain M2...M24
+        spk = parent_tx.vout[0].scriptPubKey.hex()
+        value = parent_value
+        txid = parent_txid
+        for i in range(23): # M2...M24
```

Ruff (0.14.6)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)